client/styles.jsx

```diff
@@ -0,0 +1,38 @@
+const style = {
+  timeDate: {
+    color: '#757575',
+    fontFamily: '"Roboto", sans-serif',
```

Comment:
Does it still change the font when you have the single/double quotes like this? If so then it's okay to keep.

Reply:
Changed that!